### ISSUE TITLE
USACO Parser is still not working.

### ISSUE THREAD

**Issue by cplayer** [jmerle/competitive-companion#37]
The error message is:
```
content.js:1785 TypeError: Cannot read property 'textContent' of null
    at USACOTrainingProblemParser.<anonymous> (content.js:3990)
    at Generator.next (<anonymous>)
    at content.js:3964
    at new Promise (<anonymous>)
    at __awaiter (content.js:3960)
    at USACOTrainingProblemParser.parse (content.js:3976)
    at content.js:1780
    at Generator.next (<anonymous>)
    at content.js:1749
    at new Promise (<anonymous>)
```

And I found this code in source: 
```
task.setName(elem.querySelector('center > h1').textContent);
```

But in https usaco, the task name is embraced with 

```
<center><b><font size="7"></font></b></center>
```
And the diff causes the error above.

**Comment by jmerle** [comment]
Fixed in 6afcd7b, apparently problems on training.usaco.org do not share the same format (the parser was working on other problems on the same site). I'll publish a new version later today.

**Comment by jmerle** [comment]
I've just published 2.7.3 which contains the fix. It may take up to 24 hours for your browser to update the extension.
